library/src/main/java/com/alexstyl/contactstore/UriFlow.kt

```diff
@@ -6,15 +6,16 @@ import android.net.Uri
 import android.os.Handler
 import android.os.Looper
 import kotlinx.coroutines.channels.awaitClose
-import kotlinx.coroutines.channels.trySendBlocking
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.callbackFlow
 
 internal fun ContentResolver.uriFlow(uri: Uri): Flow<Unit> {
     return callbackFlow {
         val observer = object : ContentObserver(Handler(Looper.getMainLooper())) {
             override fun onChange(selfChange: Boolean) {
-                trySendBlocking(Unit)
+                if (!channel.isClosedForSend) {
```

Comment:
Nice one 👍 

Kindly change the `!` to `.not()` as that's what is used in the rest of the project. 

Happy to ship when that is sorted.

Reply:
Done